package.json

```diff
@@ -2,9 +2,12 @@
   "scripts": {
     "build": "yarn run build-css && yarn run build-js",
     "build-css": "node-sass --omit-source-map-url --include-path node_modules static/sass --output-style compressed --output static/css && postcss --no-map --use autoprefixer --replace 'static/css/**/*.css'",
-    "build-js": "webpack && yarn run copy-3rd-party-js",
+    "prebuild-js": "yarn run copy-3rd-party-libs",
```

Comment:
I'm not really a fan of this approach, I prefer being explicit. We have so many scripts here you don't really know what's happening if things are running 'automagically'